optuna/logging.py

```diff
@@ -127,18 +127,32 @@ def disable_default_handler():
 
     Example:
 
-        Stop and then resume logging to standard output.
+        Stop and then resume logging to stderr.
```

Comment:
Good catch. Thanks!
How about adding a link to https://docs.python.org/3/library/sys.html#sys.stderr?
![image](https://user-images.githubusercontent.com/3255979/75401297-0d5c7500-5945-11ea-958e-2248bd18f869.png)

```suggestion
        Stop and then resume logging to :obj:`sys.stderr`.
```